core/README.md

```diff
@@ -20,6 +21,12 @@ corepack enable
 corepack prepare pnpm@7.14.2 --activate
 ```
 
+Install [foundry](https://book.getfoundry.sh/getting-started/installation):
+
+```bash
+curl -L https://foundry.paradigm.xyz | bash
```

Comment:
Just remembered: we also need to run `foundryup` to install forge etc:
```suggestion
curl -L https://foundry.paradigm.xyz | bash
foundryup
```

Reply:
Nice catch!

Reply:
Thanks! @claravanstaden's comment in the README jogged my memory 😅